InferenceSystem/src/model_retrain_negative_samples.py

```diff
@@ -0,0 +1,46 @@
+
+import argparse
+import datetime
+import globals
+
+def main():
+    """
+    Example usage: 
+    python3 PrepareDataForPredictionExplorer.py --start_time "2020-07-25 19:15" --end_time "2020-07-25 20:15" --dataset_folder /Users/prakrutigogia/Documents/Microsoft/AlwaysBeLearning/MSHack/Round4
```

Comment:
Huh, what about this `dataset_folder` argument?

Reply:
dataset_folder does not need to be parameter.

Reply:
addressed